content/articles/2017-12-29-dbplyr-1-2.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: dbplyr 1.2.0
+author: Edgar Ruiz
+date: '2017-12-29'
+slug: dbplyr-1-2
+categories:
+  - package
+tags: [package, tidyverse, databases]
+photo:
+  url: https://unsplash.com/photos/y7rGTFyOzxc
+  author: Giulia Bertelli
+---
+
+```{r setup, include = FALSE}
+library(dbplyr)
+library(dplyr)
+library(DBI)
+library(tibble)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are very excited to announce that [dbplyr 1.2.0](https://cran.rstudio.com/web/packages/dbplyr/index.html)  is now available on CRAN!  
+
+The dbplyr package is the database backend for dplyr. It interacts with databases directly by translating the dplyr verbs into SQL queries. Other advantages of this approach are:
```

Comment:
Since this is an update, can you please reduce the text here a bit? Ideally to one (long-ish) paragraph